rockcraft.yaml

```diff
@@ -1,12 +1,14 @@
 ---
 name: sdcore-upf-bess
-base: ubuntu@20.04
+base: ubuntu@22.04
 version: '1.3'
 summary: SD-Core UPF BESS
 description: SD-Core UPF BESS
 license: Apache-2.0
 platforms:
   amd64:
+environment:
+  LD_LIBRARY_PATH: "/usr/local/lib/x86_64-linux-gnu:/usr/local/lib"
 
 parts:
   xdp:
```

Comment:
(unrelated to this PR) Do we know if the upf works with more recent versions of `xdp`? We're using `1.2.2` which is from Jan. 2020.

Reply:
I was able to test that it builds with the latest official release, but wanted to try and keep those changes to a minimum for now. I think a next step would be to try to bump the dependencies as much as possible.